tests/test_neural_modules_export.py

```diff
@@ -0,0 +1,141 @@
+# ! /usr/bin/python
```

Comment:
can you rename this file? you are not testing Neural Module export, you are testing configuration export

Reply:
actually you are right, was thinking about that when renamed examples/tutorial